### ISSUE TITLE
Color contrast between codeblock and background negligible

### ISSUE THREAD

**Issue by thecodingaviator** [freeCodeCamp/freeCodeCamp#37153]
The color contrast between the background and the code block is almost negligible. I believe the original intent of having different colors is that users can differentiate between the code example and the normal text.

![image](https://user-images.githubusercontent.com/34807532/66496414-8105fe80-ead8-11e9-930e-c7afd586825c.png)

**Comment by RandellDawson** [comment]
On a related note, there used to be more space between the lines or `code` sections, so when `code` sections were stacked on top of each other, there was still some vertical space between them to avoid looking like below:
![image](https://user-images.githubusercontent.com/5313213/66497807-e70b6b00-ea71-11e9-8a3b-fa554b6490b6.png)

**Comment by ivan-nikolaievskyi** [comment]
@raisedadead can I get this one to fix?

**Comment by raisedadead** [comment]
Yes, please but please take a moment to go thru out [guidelines for contributing](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md)
